health.go

```diff
@@ -59,11 +60,18 @@ func (h HealthCheck) SetMaxConsecutiveFailures(i int) HealthCheck {
 	return h
 }
 
+// SetIgnoreHttp1xx sets ignore http 1xx on the health check.
+func (h HealthCheck) SetIgnoreHttp1xx(b bool) HealthCheck {
+	h.IgnoreHttp1xx = &b
+	return h
+}
+
 // NewDefaultHealthCheck creates a default application health check
 func NewDefaultHealthCheck() *HealthCheck {
 	portIndex := 0
 	path := ""
 	maxConsecutiveFailures := 3
+	ignoreHttp1xx := false
```

Comment:
Do we need to set it here (and use it later)? Wouldn't we just continue to apply the default by omitting the value?